engine/class_modules/sc_warrior.cpp

```diff
@@ -5215,18 +5224,19 @@ void warrior_t::apl_prot()
     prot->add_action( "potion,if=target.time_to_die<25" );
   }
 
-  prot->add_action( this, "Battle Cry", "if=cooldown.shield_slam.remains=0" );
-  prot->add_action( this, "Avatar" );
+  prot->add_action( this, "Avatar", "if=cooldown.demoralizing_shout.remains>5" );
   prot->add_action( this, "Demoralizing Shout" );
   prot->add_action( this, "Ravager", "if=talent.ravager.enabled" );
+  prot->add_action( this, "Dragon Roar", "if=talent.dragon_roar.enabled" );
```

Comment:
Dragon Roar is a talent so you have to use add_talent() instead of add_action(). You also don't need to specify that the spell is talented.
```suggestion
  prot->add_talent( this, "Dragon Roar" );
```